base/sparse/cholmod.jl

```diff
@@ -418,6 +418,13 @@ function copy_dense{Tv<:VTypes}(A::Dense{Tv})
     d
 end
 
+function sort!{Tv<:VTypes}(S::Sparse{Tv})
```

Comment:
To elaborate. I think this works and isn't confusing since inside `CHOLMOD` the names match the behavior of CHOLMOD. If it is important/necessary to change this it should be a separate issue but I don't think it is.